extensions/amp-a4a/amp-a4a-format.md

```diff
@@ -0,0 +1,429 @@
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+# AMP A4A AD CREATIVE FORMAT
+
+** >> WORK IN PROGRESS.  SUBJECT TO CHANGE. << **
+
+_This set of standards is still in development and is likely to be revised.
+Feedback from the community is welcome.  Please comment here or on the [Intent
+to Implement](https://github.com/ampproject/amphtml/issues/4264)_.
+
+A4A (AMP Ads for AMPHTML Pages) is a mechanism for rendering fast, performant
+ads in AMP pages.  To ensure that A4A ad documents ("A4A creatives") can be
+rendered quickly and smoothly in the browser and do not degrade user experience,
+A4A creatives must obey a set of validation rules.  Similar in spirit to the
+[AMP format rules](../../spec/amp-html-format.md), A4A creatives have access to
+a limited set of allowed tags, capabilities, and extensions.
+
+## A4A Format Rules
+ 
+1. Unless otherwise specified below, the creative must obey all rules given by 
+the [AMP format rules](../../spec/amp-html-format.md), included here by 
+reference.  For example, the
+A4A [Boilerplate](amp-a4a-format.md#2) 
+deviates from the AMP standard boilerplate.
+
+  _*In addition*_:
+
+1. The creative must use `<html a4⚡>` or `<html a4amp>` as its enclosing tags.
+
+   _Rationale_: Allows validators to identify a creative document as either a 
+ general AMP doc or a restricted A4A doc and to dispatch appropriately.
+
+1. Media: Videos must not enable autoplay.
+
+  _Rationale_: Autoplay forces video content to be downloaded immediately, which
+ slows the page load.
+
+1. Media: Audio must not enable autoplay.
+
+  _Rationale_: Same as for video.
+
+1. Analytics: `<amp-analytics>` viewability tracking may only target the full-ad
+selector, via  `"visibilitySpec": { "selector": "amp-ad" }`, as defined in
+[Issue #4018](https://github.com/ampproject/amphtml/issues/4018) and
+[PR #4368](https://github.com/ampproject/amphtml/pull/4368).  In
+particular, it may not target any selectors for elements within the ad creative.
+
+  _Rationale_: In some cases, A4A may choose to render an ad creative in an 
+  iframe.  In those cases, host page analytics can only target the entire iframe anyway, and won’t have access to any finer-grained selectors.
+
+  _Example_:
+  
+  ```html
+<amp-analytics id="nestedAnalytics">
+  <script type="application/json">
+  {
+    "requests": {
+       "visibility": "https://example.com/nestedAmpAnalytics"
+    },
+    "triggers": {
+      "visibilitySpec": {
+        "selector": "amp-ad",
+        "visiblePercentageMin": 50,
+        "continuousTimeMin": 1000
+      }
+    }
+  }
+  </script>
+</amp-analytics>
```

Comment:
If you need to add a description for this example, it would be something like:

The configuration sends a request to URL `https://example.com/nestedAmpAnalytics`  when 50% of the ad has been continuously visible on the screen for 1 second.